frontend/pages/hosts/details/cards/HostSoftwareLibrary/HostInstallerActionCell/HostInstallerActionCell.tsx

```diff
@@ -233,7 +233,18 @@ export const HostInstallerActionCell = ({
     uninstall: getInstallerActionButtonConfig("uninstall", ui_status),
   });
 
+  // Local “clicked” state so the button disables immediately
+  const [
+    isInstallUninstallPendingLocal,
+    setIsInstallUninstallPendingLocal,
+  ] = useState(false);
+
   useEffect(() => {
+    // reset local pending state when status leaves pending (API round‑trip finished)
```

Comment:
This works because status/UI status don't change as a result of anything other than an API response bubbling down into the component right?

Reply:
Yeah, those are read-only props that come from the API, but we need to ensure statuses are reset after it finishes so a action button can be clicked again once done with the initial action